include/aspect/material_model/averaging.h

```diff
@@ -0,0 +1,148 @@
+/*
+  Copyright (C) 2014 by the authors of the ASPECT code.
```

Comment:
The date is wrong. Please also just use the same formatting as we do in the other files. Just copy the whole block from somewhere else.